internal/service/component.go

```diff
@@ -0,0 +1,50 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"github.com/briandowns/spinner"
+	"github.com/dream11/odin/pkg/constant"
+	operate "github.com/dream11/odin/proto/gen/go/dream11/od/operate/v1"
+	log "github.com/sirupsen/logrus"
+	"io"
+)
+
+// Component performs operation on component like operate
+type Component struct{}
+
+// OperateComponent operate Component
+func (e *Component) OperateComponent(ctx *context.Context, request *operate.OperateServiceRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := operate.NewServiceServiceClient(conn)
+	stream, err := client.OperateService(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Operate Component in progress...")
```

Comment:
change the message here to `Operating component...` or `Starting component operation`. The message in the spinner may also be `Operate component in progress` and it doesn't look good for both to the same sentence